graphs/energy/edges/buildings/buildings_space_heater_heatpump_air_water_electricity_aggregator-buildings_useful_demand_for_space_heating_after_insulation@useable_heat.ad

```diff
@@ -0,0 +1,2 @@
+- type = share
```

Comment:
I think this edge should go to `buildings_useful_demand_for_space_heating_after_insulation_and_solar_thermal` instead of `buildings_useful_demand_for_space_heating_after_insulation`

Reply:
Yes! Good one, I've changed the edge 👍